### DISCUSSION TITLE
PL support for GCP instances with Local SSD

### DISCUSSION THREAD

**Discussion by jshin49** [Lightning-AI/pytorch-lightning#10453] [Idea pool]
https://cloud.google.com/compute/docs/disks/local-ssd

Google Cloud Platform supports Local SSD, and this supposedly significantly boosts IOPS, which could drastically affect training/testing if there's frequent CPU-GPU communication.

However, I think this may require some Pytorch or Pytorch-lightning level support.

Do you guys have any pointers I can refer to? Or please let me know if I'm wrong.

**Comment by kaushikb11** [comment]
> I think this may require some Pytorch or Pytorch-lightning level support.

Where do you think the support would be required? Have you tried it out with Lightning?

**Reply by jshin49** [reply]
when I tried using the same code (using lightning) and data on local-ssd enabled GCP instances, there were no notable changes. Meanwhile, this same piece of code would run faster on an on-premise device (which obviously has a dedicated local SSD).
